File: Android_Zone_Test/src/com/example/mylib_test/activity/touch/ViewDragStudyActivity.java
package com.example.mylib_test.activity.touch;

import android.view.View;

import com.example.mylib_test.R;
import com.example.mylib_test.activity.touch.view.ViewDragStudyFrame;

import com.zone.lib.base.activity.BaseActivity;
import com.zone.lib.base.activity.kinds.SwipeBackKind;
import com.zone.lib.utils.activity_fragment_ui.ToastUtils;

/**
 * Created by Zone on 2016/1/29.
 */
public class ViewDragStudyActivity extends BaseActivity {
    @Override
    public void setContentView() {
        setContentView(R.layout.a_viewdragstudy);
    }

    @Override
    public void updateKinds() {
        super.updateKinds();
        mKindControl.remove(SwipeBackKind.class);
    }

    @Override
    public void findIDs() {

    }

    @Override
    public void initData() {

    }

    @Override
    public void setListener() {

    }

    @Override
    public void onClick(View v) {
        super.onClick(v);
        switch (v.getId()){
          case R.id.tv_back:
              ((ViewDragStudyFrame)findViewById(R.id.viewDragStudyFrame)).toggle();
            break;
          case R.id.tv_mMenuView:
              ToastUtils.showLong(this,"没有tryCaptureView 可以点击");
            break;
        }
    }
}

File: Android_Zone_Test/src/com/example/mylib_test/activity/touch/ViewDragStudyActivity.kt
package com.example.mylib_test.activity.touch

import android.view.View

import com.example.mylib_test.R
import com.example.mylib_test.activity.touch.view.ViewDragStudyFrame
import com.zone.lib.base.controller.activity.BaseFeatureActivity

import com.zone.lib.base.controller.activity.controller.SwipeBackActivityController
import com.zone.lib.utils.activity_fragment_ui.ToastUtils

/**
 * Created by Zone on 2016/1/29.
 */
class ViewDragStudyActivity : BaseFeatureActivity() {
    override fun initBaseControllers() {
        super.initBaseControllers()
        unRegisterPrestener(SwipeBackActivityController::class.java)
    }

    override fun setContentView() {
        setContentView(R.layout.a_viewdragstudy)
    }

    override fun initData() {

    }

    override fun setListener() {

    }

    override fun onClick(v: View?) {
        super.onClick(v)
        when (v?.id) {
            R.id.tv_back -> (findViewById<View>(R.id.viewDragStudyFrame) as ViewDragStudyFrame).toggle()
            R.id.tv_mMenuView -> ToastUtils.showLong(this, "没有tryCaptureView 可以点击")
            else -> {
            }
        }
    }
}
